src/instructlab/model/evaluate.py

```diff
@@ -113,19 +111,13 @@ def get_evaluator(
 
     # ensure knowledge benchmarks have proper arguments if selected
     if benchmark in [Benchmark.MMLU, Benchmark.MMLU_BRANCH]:
-        required_args = [model, few_shots, batch_size]
-        required_arg_names = ["model"]
```

Comment:
btw probably there's a bug here, see that few_shots and batch_size are NOT listed in required_arg_names. But I left it as-is.